core/common/src/main/java/alluxio/wire/ThriftUtils.java

```diff
@@ -122,6 +124,31 @@ public static WorkerNetAddress fromThrift(alluxio.thrift.WorkerNetAddress worker
   }
 
   /**
+   * Converts thrift type to wire type.
+   *
+   * @param tTtlAction {@link TTtlAction}
+   * @return {@link TtlAction} equivalent
+   */
+  public static TtlAction fromThrift(TTtlAction tTtlAction) {
```

Comment:
this should use a TtlAction constructor (that takes the thirft TTL action) ... similar to other wire classes